src/app/search/search-filters/simple.tsx

```diff
@@ -6,6 +6,12 @@ import { FilterDefinition } from '../filter-types';
 
 // simple checks against check an attribute found on DimItem
 const simpleFilters: FilterDefinition[] = [
+  {
+    keywords: 'adept',
+    description: tl('Filter.Adept'),
```

Comment:
With this, the search filter description is literally `\(Adept\)`. This should probably be a new localization string, see https://github.com/DestinyItemManager/DIM/blob/master/docs/CONTRIBUTING.md#development -> Translation

Reply:
In order to keep existing logic working for Adept and Timelost weapons without breaking/large refactoring, I took the liberty of updating the localization strings for `Filter.Adept`, `Filter.Harrowed`, and `Filter.Timelost`.

These are now defined as such in the `Filters` section of the `i18n.json` file:
```js
"Adept": {
  "Description": "Show Adept weapons.",
  "Tag": "(Adept)"
},
"Harrowed": {
  "Description": "Show Harrowed weapons from the King's Fall raid.",
  "Tag": "(Harrowed)"
},
"Timelost": {
  "Description": "Show Timelost weapons from the Vault of Glass raid.",
  "Tag": "(Timelost)"
},
```

Reply:
I feel like a single `is:adept` filter covering Timelost and Harrowed weapons too is okay -- users can filter down further with `source:kingsfall` etc.

Also I'm not opposed to removing the RegExp syntax from the localization files, but `stripAdept` then must not create a regular expression directly from those localized strings, since otherwise the parentheses are interpreted as RegExp syntax (note how they were previously preceded by escaping backslashes).